src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommandParser.cs

```diff
@@ -21,10 +21,10 @@ internal static class WorkloadInstallCommandParser
                 ArgumentHelpName = LocalizableStrings.ConfigFileOptionName
             };
 
-        public static readonly Option AddSourceOption =
-            new Option<string[]>("--add-source", LocalizableStrings.AddSourceOptionDescription)
+        public static readonly Option SourceOption =
+            new Option<string[]>(new string[] { "-s", "--source" }, LocalizableStrings.SourceOptionDescription)
```

Comment:
If it only accepts a single source, should we change this type to `string` instead of `string[]` so the parser enforces this? With the current config it would allow `--source foo --source bar`- would that be valid?

Reply:
no, it accepts a list of sources. and use that set of sources exclusively.

--source foo --source bar is allowed. we have that for dotnet restore a long time